lib/biz/time.rb

```diff
@@ -14,8 +14,8 @@ class Time
     DAY_MINUTES  = DAY_HOURS * HOUR_MINUTES
     WEEK_MINUTES = WEEK_DAYS * DAY_MINUTES
 
-    BIG_BANG   = ::Time.new(-10**100).freeze
-    HEAT_DEATH = ::Time.new(10**100).freeze
+    BIG_BANG   = ::Clavius::Time::BIG_BANG.to_time.freeze
+    HEAT_DEATH = ::Clavius::Time::HEAT_DEATH.to_time.freeze
```

Comment:
I've added it as a to-do for the `v2.0` release of `biz` to add public methods to access `clavius`'s endpoint values and then to consume those in `biz`.

In the meantime, let's use `Time.new(-100_000_000)` and `Time.new(100_000_000)` here.